### DISCUSSION TITLE
How to get or make IsisDlm for IDL

### DISCUSSION THREAD

**Discussion by jj5406** [DOI-USGS/ISIS3#5404] [General]
I am just returning to linux system management after many years, so I apologize for not knowing/remembering much.  On an older computer (running something like centos 6), we have a user running IDL code that calls ISIS.  It seems to do this by way of a library called IsisDLm.  I installed ISIS on the new computer (running rocky 8) using anaconda (which seemed to work), but still have no idea how to get or create the required IsisDlm library.  Can anyone point me in the right direction (or point me to a better forum to ask in)?  Thanks.

**Comment by thareUSGS** [comment]
Unfortunately, IsisDlm is no longer supported (since ~2019). You can see some discussion here for it (including a script for loading raw BSQ cubes): [https://github.com/DOI-USGS/ISIS3/issues/5172](https://github.com/DOI-USGS/ISIS3/issues/5172)
- note the GDAL tips on that page too.

Now I have been working with NV5 (previously Harris) and they have stated that will turn on support for both ISIS cubes and PDS4 images in both IDL and ENVI. Since IDL/ENVI uses GDAL, they just need to enable the format. It actually may have been turned on in the latest release (untested though).

From this site, I went ahead and attached that IDL script: https://www.ipgp.fr/~lucas/isis2envi.html
[isis2envi.csh.txt](https://github.com/DOI-USGS/ISIS3/files/14542807/isis2envi.csh.txt)
- thanks to A. Lucas for providing it (license: https://www.gnu.org/licenses/gpl-3.0.en.html).
